src/libs/actions/BankAccounts.js

```diff
@@ -46,14 +46,15 @@ Onyx.connect({
 
 /**
  * Gets the Plaid Link token used to initialize the Plaid SDK
+ *
+ * @param {String} redirectURI
  */
-function fetchPlaidLinkToken() {
-    API.Plaid_GetLinkToken()
+function fetchPlaidLinkToken(redirectURI = null) {
```

Comment:
Should we ever call this without the `redirectURI`? Maybe it should not have a default?

Reply:
Yes, since from reading the Plaid OAuth docs I believe not all banking institutions will have OAuth setup.

Reply:
AH wait, I see what you're saying. Yeah, this should not have a default as we should always be passing in `null` or a `redirectURI`